src/serializers/encoder/v11/encode_schema.c

```diff
@@ -49,7 +49,18 @@ static inline void _RdbSaveIndexData(RedisModuleIO *rdb, Index *idx) {
 	RedisModule_SaveUnsigned(rdb, fields_count);
 	for(uint i = 0; i < fields_count; i++) {
 		// Indexed property
-		RedisModule_SaveStringBuffer(rdb, idx->fields[i], strlen(idx->fields[i]) + 1);
```

Comment:
At the beginning of this function `_RdbSaveIndexData` there need to be a comment similar to `_RdbSaveSchema` specifying the format / layout of an index in the RDB.